Vagrantfile

```
@@ -103,14 +105,14 @@ Vagrant.configure("2") do |config|
         end
       end
 
-      # Freebsd 
+      # Freebsd
       if node[:platform] == "freebsd"
         nodeconfig.vm.provision "shell", path: "dev-tools/vagrant_scripts/unixProvision.sh", args: "gvm amd64 freebsd #{GO_VERSION}", privileged: false
         nodeconfig.vm.provision "shell", inline: "sudo mount -t linprocfs /dev/null /proc", privileged: false
       end
 
       # gvm install
-      if node[:platform] == "centos" or node[:platform] == "ubuntu" or node[:platform] == "debian" or node[:platorm] == "archlinux"
+      if node[:platform] == "centos" or node[:platform] == "ubuntu" or node[:platform] == "debian" or node[:platform] == "archlinux" or node[:platform] == "opensuse" or node[:platform] == "sles"
```

```suggestion
      if [:centos, :ubuntu, :debian, :archlinux, :opensuse, :sles].include?(node[:platform].to_sym)
```

This statement is starting to be big...

---

I hope my ruby memory is still ok.